projects/slap_manipulation/scripts/execution.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import numpy as np
+import yaml
+
+from home_robot.motion.stretch import STRETCH_HOME_Q, HelloStretchKinematics
+from home_robot_hw.remote import StretchClient
+
+if __name__ == "__main__":
+    # load action dict from "~/actions.yaml"
+    actions = yaml.unsafe_load(open(os.path.expanduser("~/actions.yaml"), "r"))
```

Comment:
this path will be invalid - should be a relative path like projects/slap_manipulation/actions.yaml